### ISSUE TITLE
5/8 Test Kit HealthAndDV.csv issue in CurrentlyFleeing value in many records

### ISSUE THREAD

**Issue by MichelleNevius** [abtassociates/LSASampleCode#166]
Hi, Our import validator has identified dozens of records with a message like this: 
CurrentlyFleeing has to be left blank for Minnesota Shoe on their admission form dated 03/15/2018 in APR - Emergency Shelter / SSO
CurrentlyFleeing has to be left blank for Minnesota Shoe on their HMIS info record dated 04/28/2018 in APR - Emergency Shelter / SSO
Could you investigate this, please? 
If you need more examples, I can provide many. Thanks!.

**Comment by dburgstrom** [comment]
Thank you for reporting this issue. I have corrected our output and will upload a new copy.

**Comment by MichelleNevius** [comment]
Thanks, David, for all your replies. Looking forward to the new data set. I appreciate your hard work on this.

**Comment by MichelleNevius** [comment]
Hi David. Could you check again in the new 5.14 data set, please? I have even more records (most of them?) failing to import, with this as the error.  Thanks!

**Comment by dburgstrom** [comment]
Sorry about that. Fixed in latest upload.

**Comment by MichelleNevius** [comment]
Hi David, would it be possible for you to check the "-PM" data set from today, to make sure it is the proper files zipped together? I got the exact same errors I got with the set from this morning, and so it has me wondering if maybe there is a labeling issue there, or something odd going on with my importer. I retrieved the files from the "HMIS Vendors" site at LAMP Consortium

Location: site "HMIS Vendors" > Resources > Dummy Data Sets being sure to download the "pm" one.
